atlp-rwanda/cabal-bn-be
# #181339469 setting-up-unit-tests

### What does this PR do?
Set up unit tests for the project

### Description of Task to be completed?

- add packages that support testing
- add tests of welcome routes as reference

### How should this be manually tested?

- Clone this repository and open the project in your code editor.
- Check out  [chore/#181339469-set-up-unit-test](https://github.com/atlp-rwanda/cabal-bn-be/tree/chore/%23181339469-set-up-unit-test)
- to install all package on your local repo use: **yarn install** 
- to run the project use: **yarn run test**

### What are the relevant pivotal tracker stories?
[#181339469](https://www.pivotaltracker.com/n/projects/2555280)
### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/95901517/156575826-0baf1ae1-1af2-4bf9-bda6-bea1b296b2ff.png)
